README.md

```diff
@@ -13,18 +13,25 @@ of risk-limiting audits.
 
 # Installation
 
+You can install the in-development version via:
 ```
-pip install r2b2
+git clone https://github.com/gwexploratoryaudits/r2b2.git
+pip install .
```

Comment:
I agree this should be a working installation guide. For clarity, we should add the `cd r2b2` line between clone and installation.

Or, this can be reduced to 1 command:
```
pip install -e git+https://github.com/gwexploratoryaudits/r2b2.gitt#egg=r2b2
```

Reply:
Ahh - good point. I'll add the cd command.